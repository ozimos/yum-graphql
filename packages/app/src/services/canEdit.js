import differenceInMinutes from 'date-fns/difference_in_minutes';

export default (updatedAt) => {
  const orderEditMinutes = parseInt(process.env.ORDER_EDIT_MINUTES, 10) || 15;
  const minutesSinceOrder = differenceInMinutes(new Date(), updatedAt);
  return orderEditMinutes - minutesSinceOrder > 0;
};
