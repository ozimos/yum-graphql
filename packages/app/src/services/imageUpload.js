import axios from 'axios';

const imageUpload = (files, callback) => {

  const formData = new FormData();
  formData.append('file', files[0]);
  formData.append('tags', 'meals');
  formData.append('upload_preset', process.env.CLOUDINARY_UPLOAD_PRESET);
  formData.append('api_key', process.env.CLOUDINARY_API_KEY);
  formData.append('timestamp', (Date.now() / 1000) || 0);

  const config = {

    onUploadProgress(ProgressEvent) {
      let percentCompleted = 0;

      const totalLength = ProgressEvent.lengthComputable
        ? ProgressEvent.total
        : ProgressEvent.target
          .getResponseHeader('content-length')
        || ProgressEvent.target
          .getResponseHeader('x-decompressed-content-length');

      if (totalLength !== null) {
        percentCompleted = Math
          .round((ProgressEvent.loaded * 100) / totalLength);
      }

      callback(percentCompleted);
    }
  };

  return axios.post(process.env.CLOUDINARY_URL, formData, {
    headers: { 'X-Requested-With': 'XMLHttpRequest' }, ...config
  });
};
export default imageUpload;
