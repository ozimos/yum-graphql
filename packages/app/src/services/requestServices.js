import axios from 'axios';
import authHeader from './authHeader';

const requestServices = (url, requestMethod, data) => {
  const requestOptions = {
    url,
    method: requestMethod || 'get',
    headers: {
      'Content-Type': 'application/json',
      Accept: 'application/json',
      ...authHeader()
    }
  };

  if (data) {
    requestOptions.data = data;
  }
  return axios(requestOptions);
};

export default requestServices;
