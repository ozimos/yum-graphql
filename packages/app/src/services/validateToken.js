import jwt from 'jsonwebtoken';

const validateToken = (token) => {
  let valid = true;
  const decoded = jwt.decode(token);

  const { exp, isCaterer, userId, firstName } = decoded;

  if (exp < Math.floor(Date.now() / 1000)) {
    valid = false;
  }

  return { valid, isCaterer, userId, firstName };
};

export default validateToken;
