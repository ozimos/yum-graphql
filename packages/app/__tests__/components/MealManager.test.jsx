/* global React:false, shallow:false toJson:false */
import { MealManager, mapStateToProps } from
  '../../src/components/mealManager/ConnectedMealManager';


import { allMeals, meal } from '../__mocks__/mealDataMock';
import * as imageUpload from '../../src/services/imageUpload';

const props = {
  pagination: {
    limit: 8,
    offset: 1,
    pages: 1
  },
  mealError: '',
  connecting: false,
  meals: allMeals.data.rows,
  user: { isCaterer: true, firstName: 'user' },
  dispatch: jest.fn(),
};

describe('MealManager Component', () => {

  const setup = () => shallow(<MealManager {...props} />);

  it('renders correctly', () => {

    let wrapper = setup();
    expect(toJson(wrapper)).toMatchSnapshot();

    const propsWhenConnecting = { ...props, meals: [], connecting: true };
    wrapper = shallow(<MealManager {...propsWhenConnecting} />);
    expect(toJson(wrapper)).toMatchSnapshot();

    wrapper.setState({
      canSubmit: false, displayImage: true, uploading: false
    });
    expect(toJson(wrapper)).toMatchSnapshot();

  });

  it('should call `setUploadPercent` when an image is selected', () => {
    const wrapper = setup();

    const setUploadPercentSpy = jest.spyOn(
      wrapper.instance(),
      'setUploadPercent'
    );

    wrapper.instance().setUploadPercent();
    expect(setUploadPercentSpy).toHaveBeenCalled();
  });

  it(
    'should call `handleImageDrop` when the upload image button is clicked',
    () => {

      const imageUploadSpy = jest.spyOn(imageUpload, 'default');
      imageUploadSpy.mockResolvedValue({ data: { secure_url: 'url' } });
      const wrapper = setup();

      const handleImageDropSpy = jest.spyOn(
        wrapper.instance(),
        'handleImageDrop'
      );

      wrapper.instance().handleImageDrop([]);
      expect(handleImageDropSpy).toHaveBeenCalled();
    }
  );

  it(
    'should call `openNewMealModal`when the add meal button is clicked',
    () => {
      const wrapper = setup();

      const openNewMealModalSpy = jest.spyOn(
        wrapper.instance(),
        'openNewMealModal'
      );
      wrapper.instance().openNewMealModal();
      expect(openNewMealModalSpy).toHaveBeenCalled();
    }
  );

  it('should call `closeNewMealModal` when the close button is clicked', () => {

    const wrapper = setup();

    const closeNewMealModalSpy = jest.spyOn(
      wrapper.instance(),
      'closeNewMealModal'
    );

    wrapper.instance().closeNewMealModal();
    expect(closeNewMealModalSpy).toHaveBeenCalled();
  });

  it('should call `disableButton`', () => {
    const wrapper = setup();

    const disableButtonSpy = jest.spyOn(
      wrapper.instance(),
      'disableButton'
    );
    wrapper.instance().disableButton();
    expect(disableButtonSpy).toHaveBeenCalled();
  });

  it('should call `enableButton`', () => {
    const wrapper = setup();

    const enableButtonSpy = jest.spyOn(
      wrapper.instance(),
      'enableButton'
    );
    wrapper.instance().enableButton();
    expect(enableButtonSpy).toHaveBeenCalled();
  });

  it('should call `searchUpdated`', () => {
    const wrapper = setup();

    const searchUpdatedSpy = jest.spyOn(
      wrapper.instance(),
      'searchUpdated'
    );
    wrapper.instance().searchUpdated('allow');
    expect(searchUpdatedSpy).toHaveBeenCalled();
  });

  it('should call `handlePaginationClick`', () => {
    const wrapper = setup();

    const handlePaginationClickSpy = jest.spyOn(
      wrapper.instance(),
      'handlePaginationClick'
    );
    wrapper.instance().handlePaginationClick({ selected: 1 });
    expect(handlePaginationClickSpy).toHaveBeenCalled();
  });

  it('should call `handleCreateMeal`', () => {
    const wrapper = setup();

    const handleCreateMealSpy = jest.spyOn(
      wrapper.instance(),
      'handleCreateMeal'
    );
    wrapper.instance().handleCreateMeal(meal.data);
    expect(handleCreateMealSpy).toHaveBeenCalled();
  });

  it('should map state to props', () => {
    const initialState = {
      loginReducer: {
        authenticated: true,
        user: {
          isCaterer: true
        }
      },
      mealsReducer: {
        mealError: '',
        pagination: {
          limit: 10,
          offset: 0
        },
        connecting: false,
        meals: []
      }
    };

    const tree = mapStateToProps(initialState);
    expect(tree).toMatchSnapshot();
  });
});
