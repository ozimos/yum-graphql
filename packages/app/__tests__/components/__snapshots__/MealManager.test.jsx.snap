// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MealManager Component renders correctly 1`] = `
<div
  className="contain"
>
  <header
    className="header"
  >
    <Connect(Nav) />
  </header>
  <main
    className="min-height"
  >
    <Greeting
      firstName="user"
      isCaterer={true}
    />
    <div
      className="title-element flexbox"
    >
      <h5
        className="shrink"
      >
        Your Meals
      </h5>
      <button
        className="btn title-button"
        onClick={[Function]}
      >
        <p>
          Add Meal
        </p>
      </button>
    </div>
    <React.Fragment>
      <Search
        caseSensitive={false}
        className="search-input input-field"
        fuzzy={false}
        onChange={[Function]}
        throttle={200}
      />
      <MealCardContainer
        MealCard={[Function]}
        addClass="scroll2"
        connecting={false}
        meals={
          Array [
            Object {
              "description": "nice",
              "id": "meal1",
              "imageUrl": "nice",
              "price": 1200,
              "title": "Eba and Egusi",
              "userId": "user1",
            },
            Object {
              "description": "nice",
              "id": "meal2",
              "imageUrl": "nice",
              "price": 2000,
              "title": "Rice and Stew",
              "userId": "user1",
            },
          ]
        }
      />
      <PaginationBoxView
        activeClassName="active"
        breakLabel="..."
        containerClassName="pagination"
        disableInitialCallback={false}
        disabledClassName="disabled"
        marginPagesDisplayed={2}
        nextClassName="next"
        nextLabel="next"
        onPageChange={[Function]}
        pageCount={1}
        pageRangeDisplayed={5}
        previousClassName="previous"
        previousLabel="previous"
        subContainerClassName="pages pagination"
      />
    </React.Fragment>
  </main>
  <Modal
    ariaHideApp={true}
    bodyOpenClassName="ReactModal__Body--open"
    className="modal-content"
    closeTimeoutMS={0}
    contentLabel="Input Modal"
    isOpen={false}
    onRequestClose={[Function]}
    parentSelector={[Function]}
    portalClassName="ReactModalPortal"
    shouldCloseOnEsc={true}
    shouldCloseOnOverlayClick={true}
    shouldFocusAfterRender={true}
    shouldReturnFocusAfterClose={true}
  >
    <div
      className="title flexbox navbar-fixed"
    >
      <h4
        className="shrink"
      >
        Meal Editor
      </h4>
      <button
        className="btn title-button"
        onClick={[Function]}
      >
        ✖
      </button>
    </div>
    <div
      className="form3-grid"
    >
      <div>
        <Formsy
          className="form3"
          disabled={false}
          getErrorMessage={[Function]}
          getErrorMessages={[Function]}
          getValue={[Function]}
          hasValue={[Function]}
          isFormDisabled={[Function]}
          isFormSubmitted={[Function]}
          isPristine={[Function]}
          isRequired={[Function]}
          isValid={[Function]}
          isValidValue={[Function]}
          mapping={null}
          onChange={[Function]}
          onError={[Function]}
          onInvalid={[Function]}
          onInvalidSubmit={[Function]}
          onReset={[Function]}
          onSubmit={[Function]}
          onValid={[Function]}
          onValidSubmit={[Function]}
          preventExternalInvalidation={false}
          resetValue={[Function]}
          setValidations={[Function]}
          setValue={[Function]}
          showError={[Function]}
          showRequired={[Function]}
          validationErrors={null}
        >
          <Formsy(Input)
            innerRef={[Function]}
            name="title"
            placeholder="Meal Title"
            required={true}
            typeOfInput="text"
            validationError="Please enter the meal title"
            validationErrors={
              Object {
                "maxLength": "input must be shorter than 50 characters",
                "minLength": "input must be longer than 1 character",
              }
            }
            validations={
              Object {
                "maxLength": 48,
                "minLength": 1,
              }
            }
          />
          <Formsy(Input)
            innerRef={[Function]}
            name="price"
            placeholder="Price"
            required={true}
            typeOfInput="number"
            validationError="Please enter a price"
            validationErrors={
              Object {
                "isOnlyInt": "price must be integer",
                "maxLength": "input must be shorter than 50 characters",
                "minLength": "input must be longer than 1 character",
              }
            }
            validations={
              Object {
                "isOnlyInt": [Function],
                "maxLength": 48,
                "minLength": 1,
              }
            }
          />
          <Formsy(TextArea)
            innerRef={[Function]}
            name="description"
            placeholder="Description"
            required={true}
            validationError=""
            validationErrors={
              Object {
                "maxLength": "input must be shorter than 100 characters",
                "minLength": "input must be longer than 5 characters",
              }
            }
            validations={
              Object {
                "maxLength": 98,
                "minLength": 5,
              }
            }
          />
          <Formsy(Input)
            innerRef={[Function]}
            name="imageUrl"
            required={true}
            style={
              Object {
                "height": 0,
                "margin": 0,
                "visibility": "hidden",
              }
            }
            typeOfInput="url"
            validationError="Please select an image"
            validationErrors={
              Object {
                "isUrl": "A valid url was not supplied",
                "maxLength": "input must be shorter than 150 characters",
                "minLength": "input must be longer than 5 characters",
              }
            }
            validations={
              Object {
                "isUrl": true,
                "maxLength": 148,
                "minLength": 5,
              }
            }
            value="https://res.cloudinary.com/tovieyeozim/image/upload/c_fill,w_200,h_200/v1532706744/hbj4qsguzdi78xcksws8.jpg"
          />
        </Formsy>
      </div>
      <div
        className="overlay-container"
      >
        <div
          className="overlay full"
        >
          <t
            accept="image/*"
            className="dropzone"
            disableClick={false}
            disablePreview={false}
            disabled={false}
            maxSize={Infinity}
            minSize={0}
            multiple={true}
            onDrop={[Function]}
            preventDropOnDocument={true}
          >
            <button
              className="btn"
            >
               Select an Image
            </button>
          </t>
        </div>
        <div
          id="meal_image"
        >
          <img
            alt="meal"
            className="fluid-img"
            src="https://res.cloudinary.com/tovieyeozim/image/upload/c_fill,w_200,h_200/v1532706744/hbj4qsguzdi78xcksws8.jpg"
          />
        </div>
      </div>
      <button
        className="btn btn-disabled"
        disabled={true}
        onClick={[Function]}
        type="submit"
      >
        <p>
          Continue
        </p>
      </button>
    </div>
  </Modal>
</div>
`;

exports[`MealManager Component renders correctly 2`] = `
<div
  className="contain"
>
  <header
    className="header"
  >
    <Connect(Nav) />
  </header>
  <main
    className="min-height"
  >
    <Greeting
      firstName="user"
      isCaterer={true}
    />
    <div
      className="title-element flexbox"
    >
      <h5
        className="shrink"
      >
        Your Meals
      </h5>
      <button
        className="btn title-button"
        onClick={[Function]}
      >
        <p>
          Add Meal
        </p>
      </button>
    </div>
    <div>
      <h5>
        You have not added any meals. Get started by clicking the add a meal button
      </h5>
    </div>
  </main>
  <Modal
    ariaHideApp={true}
    bodyOpenClassName="ReactModal__Body--open"
    className="modal-content"
    closeTimeoutMS={0}
    contentLabel="Input Modal"
    isOpen={false}
    onRequestClose={[Function]}
    parentSelector={[Function]}
    portalClassName="ReactModalPortal"
    shouldCloseOnEsc={true}
    shouldCloseOnOverlayClick={true}
    shouldFocusAfterRender={true}
    shouldReturnFocusAfterClose={true}
  >
    <div
      className="title flexbox navbar-fixed"
    >
      <h4
        className="shrink"
      >
        Meal Editor
      </h4>
      <button
        className="btn title-button"
        onClick={[Function]}
      >
        ✖
      </button>
    </div>
    <div
      className="form3-grid"
    >
      <div>
        <Formsy
          className="form3"
          disabled={false}
          getErrorMessage={[Function]}
          getErrorMessages={[Function]}
          getValue={[Function]}
          hasValue={[Function]}
          isFormDisabled={[Function]}
          isFormSubmitted={[Function]}
          isPristine={[Function]}
          isRequired={[Function]}
          isValid={[Function]}
          isValidValue={[Function]}
          mapping={null}
          onChange={[Function]}
          onError={[Function]}
          onInvalid={[Function]}
          onInvalidSubmit={[Function]}
          onReset={[Function]}
          onSubmit={[Function]}
          onValid={[Function]}
          onValidSubmit={[Function]}
          preventExternalInvalidation={false}
          resetValue={[Function]}
          setValidations={[Function]}
          setValue={[Function]}
          showError={[Function]}
          showRequired={[Function]}
          validationErrors={null}
        >
          <Formsy(Input)
            innerRef={[Function]}
            name="title"
            placeholder="Meal Title"
            required={true}
            typeOfInput="text"
            validationError="Please enter the meal title"
            validationErrors={
              Object {
                "maxLength": "input must be shorter than 50 characters",
                "minLength": "input must be longer than 1 character",
              }
            }
            validations={
              Object {
                "maxLength": 48,
                "minLength": 1,
              }
            }
          />
          <Formsy(Input)
            innerRef={[Function]}
            name="price"
            placeholder="Price"
            required={true}
            typeOfInput="number"
            validationError="Please enter a price"
            validationErrors={
              Object {
                "isOnlyInt": "price must be integer",
                "maxLength": "input must be shorter than 50 characters",
                "minLength": "input must be longer than 1 character",
              }
            }
            validations={
              Object {
                "isOnlyInt": [Function],
                "maxLength": 48,
                "minLength": 1,
              }
            }
          />
          <Formsy(TextArea)
            innerRef={[Function]}
            name="description"
            placeholder="Description"
            required={true}
            validationError=""
            validationErrors={
              Object {
                "maxLength": "input must be shorter than 100 characters",
                "minLength": "input must be longer than 5 characters",
              }
            }
            validations={
              Object {
                "maxLength": 98,
                "minLength": 5,
              }
            }
          />
          <Formsy(Input)
            innerRef={[Function]}
            name="imageUrl"
            required={true}
            style={
              Object {
                "height": 0,
                "margin": 0,
                "visibility": "hidden",
              }
            }
            typeOfInput="url"
            validationError="Please select an image"
            validationErrors={
              Object {
                "isUrl": "A valid url was not supplied",
                "maxLength": "input must be shorter than 150 characters",
                "minLength": "input must be longer than 5 characters",
              }
            }
            validations={
              Object {
                "isUrl": true,
                "maxLength": 148,
                "minLength": 5,
              }
            }
            value="https://res.cloudinary.com/tovieyeozim/image/upload/c_fill,w_200,h_200/v1532706744/hbj4qsguzdi78xcksws8.jpg"
          />
        </Formsy>
      </div>
      <div
        className="overlay-container"
      >
        <div
          className="overlay full"
        >
          <t
            accept="image/*"
            className="dropzone"
            disableClick={false}
            disablePreview={false}
            disabled={false}
            maxSize={Infinity}
            minSize={0}
            multiple={true}
            onDrop={[Function]}
            preventDropOnDocument={true}
          >
            <button
              className="btn"
            >
               Select an Image
            </button>
          </t>
        </div>
        <div
          id="meal_image"
        >
          <img
            alt="meal"
            className="fluid-img"
            src="https://res.cloudinary.com/tovieyeozim/image/upload/c_fill,w_200,h_200/v1532706744/hbj4qsguzdi78xcksws8.jpg"
          />
        </div>
      </div>
      <WithStyles(LinearProgress)
        style={
          Object {
            "height": "10px",
          }
        }
      />
    </div>
  </Modal>
</div>
`;

exports[`MealManager Component renders correctly 3`] = `
<div
  className="contain"
>
  <header
    className="header"
  >
    <Connect(Nav) />
  </header>
  <main
    className="min-height"
  >
    <Greeting
      firstName="user"
      isCaterer={true}
    />
    <div
      className="title-element flexbox"
    >
      <h5
        className="shrink"
      >
        Your Meals
      </h5>
      <button
        className="btn title-button"
        onClick={[Function]}
      >
        <p>
          Add Meal
        </p>
      </button>
    </div>
    <div>
      <h5>
        You have not added any meals. Get started by clicking the add a meal button
      </h5>
    </div>
  </main>
  <Modal
    ariaHideApp={true}
    bodyOpenClassName="ReactModal__Body--open"
    className="modal-content"
    closeTimeoutMS={0}
    contentLabel="Input Modal"
    isOpen={false}
    onRequestClose={[Function]}
    parentSelector={[Function]}
    portalClassName="ReactModalPortal"
    shouldCloseOnEsc={true}
    shouldCloseOnOverlayClick={true}
    shouldFocusAfterRender={true}
    shouldReturnFocusAfterClose={true}
  >
    <div
      className="title flexbox navbar-fixed"
    >
      <h4
        className="shrink"
      >
        Meal Editor
      </h4>
      <button
        className="btn title-button"
        onClick={[Function]}
      >
        ✖
      </button>
    </div>
    <div
      className="form3-grid"
    >
      <div>
        <Formsy
          className="form3"
          disabled={false}
          getErrorMessage={[Function]}
          getErrorMessages={[Function]}
          getValue={[Function]}
          hasValue={[Function]}
          isFormDisabled={[Function]}
          isFormSubmitted={[Function]}
          isPristine={[Function]}
          isRequired={[Function]}
          isValid={[Function]}
          isValidValue={[Function]}
          mapping={null}
          onChange={[Function]}
          onError={[Function]}
          onInvalid={[Function]}
          onInvalidSubmit={[Function]}
          onReset={[Function]}
          onSubmit={[Function]}
          onValid={[Function]}
          onValidSubmit={[Function]}
          preventExternalInvalidation={false}
          resetValue={[Function]}
          setValidations={[Function]}
          setValue={[Function]}
          showError={[Function]}
          showRequired={[Function]}
          validationErrors={null}
        >
          <Formsy(Input)
            innerRef={[Function]}
            name="title"
            placeholder="Meal Title"
            required={true}
            typeOfInput="text"
            validationError="Please enter the meal title"
            validationErrors={
              Object {
                "maxLength": "input must be shorter than 50 characters",
                "minLength": "input must be longer than 1 character",
              }
            }
            validations={
              Object {
                "maxLength": 48,
                "minLength": 1,
              }
            }
          />
          <Formsy(Input)
            innerRef={[Function]}
            name="price"
            placeholder="Price"
            required={true}
            typeOfInput="number"
            validationError="Please enter a price"
            validationErrors={
              Object {
                "isOnlyInt": "price must be integer",
                "maxLength": "input must be shorter than 50 characters",
                "minLength": "input must be longer than 1 character",
              }
            }
            validations={
              Object {
                "isOnlyInt": [Function],
                "maxLength": 48,
                "minLength": 1,
              }
            }
          />
          <Formsy(TextArea)
            innerRef={[Function]}
            name="description"
            placeholder="Description"
            required={true}
            validationError=""
            validationErrors={
              Object {
                "maxLength": "input must be shorter than 100 characters",
                "minLength": "input must be longer than 5 characters",
              }
            }
            validations={
              Object {
                "maxLength": 98,
                "minLength": 5,
              }
            }
          />
          <Formsy(Input)
            innerRef={[Function]}
            name="imageUrl"
            required={true}
            style={
              Object {
                "height": 0,
                "margin": 0,
                "visibility": "hidden",
              }
            }
            typeOfInput="url"
            validationError="Please select an image"
            validationErrors={
              Object {
                "isUrl": "A valid url was not supplied",
                "maxLength": "input must be shorter than 150 characters",
                "minLength": "input must be longer than 5 characters",
              }
            }
            validations={
              Object {
                "isUrl": true,
                "maxLength": 148,
                "minLength": 5,
              }
            }
            value="https://res.cloudinary.com/tovieyeozim/image/upload/c_fill,w_200,h_200/v1532706744/hbj4qsguzdi78xcksws8.jpg"
          />
        </Formsy>
      </div>
      <div
        className="overlay-container"
      >
        <div
          className="overlay full"
        >
          <t
            accept="image/*"
            className="dropzone"
            disableClick={false}
            disablePreview={false}
            disabled={false}
            maxSize={Infinity}
            minSize={0}
            multiple={true}
            onDrop={[Function]}
            preventDropOnDocument={true}
          >
            <button
              className="btn"
            >
               Select an Image
            </button>
          </t>
        </div>
        <div
          id="meal_image"
        >
          <img
            alt="meal"
            className="fluid-img"
            src={true}
          />
        </div>
      </div>
      <WithStyles(LinearProgress)
        style={
          Object {
            "height": "10px",
          }
        }
      />
    </div>
  </Modal>
</div>
`;

exports[`MealManager Component should map state to props 1`] = `
Object {
  "authenticated": true,
  "connecting": false,
  "mealError": "",
  "meals": Array [],
  "pagination": Object {
    "limit": 10,
    "offset": 0,
  },
  "user": Object {
    "isCaterer": true,
  },
}
`;
